portfolio/src/main/webapp/style/styleGCG.css

```diff
@@ -1,5 +1,17 @@
-#intro {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+h1 {
+  width: 100%;
+  background-color: #34623F;
+  color: white;
+  font-size: 70px;
 }
```

Comment:
This seems to be the same across every file. In this situation, I suggest moving it to `style.css`. The files which you want this style applied to can import multiple CSS files, e.g. `styleGCG.css` and `style.css`. 

Same thing for other elements that you might be sharing the style.